spec/lib/simp/beaker_helpers_spec.rb

```diff
@@ -103,20 +103,17 @@ def initialize
     end
 
     it 'uses defaults when no environment variables are set' do
-      expected = {
-      :puppet_install_version   => Simp::BeakerHelpers::DEFAULT_PUPPET_AGENT_VERSION,
-      :beaker_puppet_collection => nil,
-      :puppet_install_type      => 'agent'
-      }
-      expect( @helper.get_puppet_install_info ).to eq expected
+      expect( @helper.get_puppet_install_info[:puppet_install_version] ).to match(/^5\./)
+      expect( @helper.get_puppet_install_info[:puppet_collection] ).to eq('puppet5')
+      expect( @helper.get_puppet_install_info[:puppet_install_type] ).to eq('agent')
     end
 
     it 'extracts info from PUPPET_INSTALL_VERSION for Puppet 4' do
       ENV['PUPPET_INSTALL_VERSION']= '4.10.5'
```

Comment:
Can't we just remove this test since p4 is past EoL? (And it is stripped from our tests in other modules as well)

Reply:
@sharkbruhaha No. We might still need to go back and test Puppet 4 for some reason via Beaker. This was the minimum that I could do to get things working again.